EIPS/eip-1901.md

```diff
@@ -0,0 +1,78 @@
+
+![MultGethRpc-usecase](https://user-images.githubusercontent.com/364566/54172981-4605f480-443d-11e9-9e29-80c580f9edae.png)
+
+# Specification
+
+### What is OpenRPC?
+The [OpenRPC](https://github.com/open-rpc/spec) Specification defines a standard, programming language-agnostic interface description for [JSON-RPC 2.0](https://www.jsonrpc.org/specification) APIs, which allows both humans and computers to discover and understand the capabilities of a service without requiring access to source code, additional documentation, or inspection of network traffic. When properly defined via OpenRPC, a consumer can understand and interact with the remote service with a minimal amount of implementation logic, and share these logic patterns across use cases. Similar to what interface descriptions have done for lower-level programming, the OpenRPC Specification removes guesswork in calling a service.
+
+##### Structure
+This is the structure of an OpenRPC Document:
+
+![openrpc-spec-structure](https://github.com/open-rpc/design/raw/master/diagrams/structure/OpenRPC_structure.png)
+
+
+# Rationale
+### Why would we do this?
+Services need to figure out how to talk to each other. If we really want to build the next generation of automation, then having up to date libraries, documented APIs, and modern tools are going to provide easy discovery, on-boarding, and enable end user and developer interaction.
+
+Use cases for machine-readable [JSON-RPC 2.0](https://www.jsonrpc.org/specification) API definition documents include, but are not limited to:
+
+- A common vocabulary and document will keep developers, testers, architects, and technical writers all in sync.
+- Server stubs/skeletons generated in many languages
+- Clients generated in many languages
+- Mock Server generated in many languages
+- Tests generated in many languages
+- Documentation Generation
+
+
+# Alternative
+[OpenRPC](https://github.com/open-rpc/spec) documents just describe [JSON-RPC](https://www.jsonrpc.org/specification) APIs services, and are represented in JSON format. These documents may be produced and served statically OR generated dynamically from an application and returned via the [`rpc.discover`](https://github.com/open-rpc/spec#service-discovery-method) method. This gives projects and communities the flexibility to adopt tools before the [`rpc.discover`](https://github.com/open-rpc/spec#service-discovery-method) method is implemented.
+
+## Resources
+
+- [EDCON 2019 talk on OpenRPC and The Future of JSON-RPC Tooling](https://www.youtube.com/watch?v=UgSPMZ9FQ4Q)
+- [open-rpc.org](https://open-rpc.org)
+- [etclabscore/ethereum-json-rpc-specification](https://github.com/etclabscore/ethereum-json-rpc-specification)
+
+## Copyright
+
+ Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/) and code via [Apache-2.0](https://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
I think there's no code in this EIP so it may be misleading presenting the license for code here.